src/main/java/com/dih/connector/test/client/connector/DataspaceConnectorConfiguration.java

```diff
@@ -15,4 +36,44 @@
     public BasicAuthRequestInterceptor adminAuth() {
         return new BasicAuthRequestInterceptor(username, password);
     }
+
+    @Bean
+    public BasicAuthenticationInterceptor authInterceptor() {
+        return new BasicAuthenticationInterceptor(username, password);
+    }
+
+    @Bean(name = "json-ld")
```

Comment:
Why do we need to use RestTemplate? We could create separate configuration for feign client only for specific interfaces (like `/api/ids`) where we could put this http message converter

Reply:
I do not use explicit model here, rather a generic JsonNode as a container for any message. IMHO RestTemplate is more appropriate here (I would use lower level HTTP client even for that, but RestTemplate is ok as well)